.env.sample

```diff
@@ -1,5 +1,6 @@
-NC_UPSTREAM_GQL_ENDPOINT=
-NC_DOWNSTREAM_GQL_ENDPOINT=
+NC_REGISTRY_ENDPOINT=
+NC_UPSTREAM_PLANET=
+NC_DOWNSTREAM_PLANET=
```

Comment:
I think it will make users hard to test in a local environment. (with local headlesses). I wonder and will follow @longfin's opinion.

Reply:
So I was originally giving just gql endpoint array on index.ts, instead of giving the whole planet, so you can add it in .env,

But I was thinking using 9cscan REST in the future, so I just did like this.

On test, I was thinking about providing fixture, but you have a point. it's bit messy.

Reply:
> But I was thinking using 9cscan REST in the future, so I just did like this.

I think it should use 9cscan when 9cscan became able to run in the local environment 👀 ⚠️

> So I was originally giving just gql endpoint array on index.ts, instead of giving the whole planet, so you can add it in .env,
> Alternatively I also considered some local registry flag, so you can fetch from local registry-schemed json when running preparePlanets(). how about that?

Well.. it can be okay when we can use the local path. Even if not, we can bypass it by running a local file server.

Reply:
`Well.. it can be okay when we can use the local path. Even if not, we can bypass it by running a local file server.`

Um, what I mean was, if just add local path in NC_REGISTRY_ENDPOINT, fetch / fs logic branching on JS side can be done by adding it to URL constructor.

Reply:
At this time.. it's okay.